Automattic/woocommerce-payments
# Update to disable enabling APMs (except card payments) for PO users

Fixes #6413

#### Changes proposed in this Pull Request





* Add `isEnabled` and `isComplete` to the `wcPaySettings` PO array.
* Use the `more information needed` error if a user is PO enabled but not yet PO complete, and prevent adding the APM until PO is complete:





#### Testing instructions





* Run `npm run listen` on the server and ensure you pull the latest `trunk`.
* Onboard a new PO account.
* Go to `Payments > Settings` and verify that you cannot add any APMs (card will already be enabled).
* The `Learn more` link shown in the screenshot should take the user to documentation about providing details to Stripe.
* Complete full verification via the `Payments > Overview` page.
* Go back to `Payments > Settings` and verify you can add APMs.

-------------------


**Post merge**
